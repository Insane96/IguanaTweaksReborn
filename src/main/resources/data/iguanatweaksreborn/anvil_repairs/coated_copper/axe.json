{
	"item_to_repair": {
		"id": "iguanatweaksreborn:coated_copper_axe"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:obsidian"
			},
			"amount": 3
		}
	]
}