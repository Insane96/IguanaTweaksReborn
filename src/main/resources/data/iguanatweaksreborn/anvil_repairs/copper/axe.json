{
	"item_to_repair": {
		"id": "iguanatweaksreborn:copper_axe"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:copper_ingot"
			},
			"amount": 3
		}
	]
}