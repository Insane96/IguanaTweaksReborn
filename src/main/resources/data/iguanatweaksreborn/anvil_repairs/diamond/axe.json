{
	"item_to_repair": {
		"id": "minecraft:diamond_axe"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:diamond"
			},
			"amount": 2
		}
	]
}