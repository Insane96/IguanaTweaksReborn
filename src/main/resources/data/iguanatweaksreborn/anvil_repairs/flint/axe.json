{
	"item_to_repair": {
		"id": "iguanatweaksreborn:flint_axe"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:flint"
			},
			"amount": 3
		}
	]
}