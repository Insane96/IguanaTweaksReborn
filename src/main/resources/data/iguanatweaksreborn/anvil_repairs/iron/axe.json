{
	"item_to_repair": {
		"id": "minecraft:iron_axe"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:iron_ingot"
			},
			"amount": 2
		},
		{
			"repair_material": {
				"id": "minecraft:iron_nugget"
			},
			"amount": 22,
			"cost_multiplier": 9
		}
	]
}