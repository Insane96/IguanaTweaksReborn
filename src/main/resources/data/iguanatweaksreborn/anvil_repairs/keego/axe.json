{
	"item_to_repair": {
		"id": "iguanatweaksreborn:keego_axe"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:diamond"
			},
			"amount": 3,
			"max_repair": 0.5
		},
		{
			"repair_material": {
				"id": "iguanatweaksreborn:keego"
			},
			"amount": 2
		}
	]
}