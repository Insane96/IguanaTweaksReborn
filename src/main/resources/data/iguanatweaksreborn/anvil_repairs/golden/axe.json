{
	"item_to_repair": {
		"id": "minecraft:golden_axe"
	},
	"repair": [
		{
			"repair_material": {
				"id": "minecraft:gold_ingot"
			},
			"amount": 2
		},
		{
			"repair_material": {
				"id": "minecraft:gold_nugget"
			},
			"amount": 22,
			"cost_multiplier": 9
		}
	]
}